isb_lib/geome_adapter.py

```diff
@@ -357,3 +359,8 @@ def reloadThing(thing):
     L.debug("reloadThing id=%s", thing.id)
     identifier = igsn_lib.normalize(thing.id)
     return loadThing(identifier, thing.tcreated)
+
+def reparseAsCoreRecord(thing: igsn_lib.models.thing.Thing) -> typing.Dict:
+    _validateResolvedContent(thing)
+    transformer = isamples_metadata.GEOMETransformer.GEOMETransformer(thing.resolved_content)
+    return isb_lib.core.coreRecordAsSolrDoc(transformer.transform())
```

Comment:
Other than the new fields, this ended up being pretty much the totality of what was needed to get GEOME into solr.  It should be pretty easy for the other collections as well, since we're transforming the iSamples core metadata to solr.